operator/src/main/resources/scripts/model_wdt_mii_filter.py

```diff
@@ -318,14 +318,19 @@ def customizeServers(model):
 
 def customizeServer(model, server, name):
   listen_address=env.toDNS1123Legal(env.getDomainUID() + "-" + name)
-  adminServer = model['topology']['AdminServerName']
+  try:
+    adminServer = model['topology']['AdminServerName']
+  except KeyError:
+    adminServer = None
```

Comment:
Why not just use an if test instead of relying on an exception?

```
adminServer = None
if 'AdminServerName' in model['topology'] and len(model['topology']['AdminServerName']) > 0:
    adminServer = model['topology']['AdminServerName']
```

Reply:
Changed to use the `if` test as suggested. thanks.